addons/binding/org.openhab.binding.toon/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You might want to define some (>50%?) of your channels as "advanced", so that the UI is by default not too cluttered.